tensorflow-logging/tensorflow-logging.cabal

```diff
@@ -29,10 +29,10 @@ library
                 , stm
                 , stm-chans
                 , stm-conduit
-                , tensorflow == 0.1.*
-                , tensorflow-core-ops == 0.1.*
-                , tensorflow-ops == 0.1.*
-                , tensorflow-proto == 0.1.*
+                , tensorflow == 0.2.*
+                , tensorflow-core-ops == 0.2.*
+                , tensorflow-ops == 0.2.*
+                , tensorflow-proto == 0.2.*
                 , tensorflow-records-conduit
```

Comment:
While we're here, add an explicit version for tensorflow-records-conduit?

Reply:
Done.